netbout-web/src/main/java/com/netbout/rest/TkAppAuth.java

```diff
@@ -81,30 +88,46 @@
      * @param pass Last Pass on Chain
      */
     TkAppAuth(final Take take, final Pass pass) {
-        super(TkAppAuth.make(take, pass));
+        super(TkAppAuth.make(take, pass, TkAppAuth.BASICAUTH));
+    }
+    /**
+     * Ctor.
+     * @param take Take
+     * @param pass Last Pass on Chain
+     * @param basic Use Basic Auth
+     */
+    TkAppAuth(final Take take, final Pass pass, final boolean basic) {
+        super(TkAppAuth.make(take, pass, basic));
     }
 
     /**
      * Authenticated.
      * @param take Take
      * @param pass Last Pass on Chain
+     * @param basic Use Basic Auth
      * @return Authenticated take
-     * @todo #685:30m/DEV Need to implement PsBasic.Entry.
-     *  PsFake usage should be changed to
-     *  PsBasic(
-     *  Manifests.read("Netbout-Basic-Realm"),
-     *  new PsBasic.Default(
-     *  Manifests.read("Netbout-Basic-User"),
-     *  Manifests.read("Netbout-Basic-Pwd")))
-     *  But PsBasic.Default like class is currently missing. See takes #349.
-     *  It is also needed to be able to switch on/off BasicAuth e.g.
-     *  by via change the manifest entry "Netbout-Basic":true/false.
      */
-    private static Take make(final Take take, final Pass pass) {
+    private static Take make(final Take take, final Pass pass,
+        final boolean basic) {
+        final Pass auth;
+        if (basic) {
+            auth = new PsBasic(
+                Manifests.read("Netbout-Basic-Realm"),
+                new PsBasic.Default(
+                    String.format(
+                        "%s %s urn:foo:z",
```

Comment:
@pinaf yeah, `urn` never be used in this case. `PsTwice` always returns `urn' from the second argument.

https://github.com/yegor256/netbout/blob/master/netbout-web/src/main/java/com/netbout/rest/PsTwice.java
>  Sequentially tries first and second Pass instances. Identity of a second
  Pass is returned if first passes. Empty identity is returned otherwise.